savoirfairelinux/sous-chef
# Command to set orders status to Delivered

## Fixes #822

### Changes proposed in this pull request:
* src/order/management/commands/setordersdelivered.py : new command
* src/order/models.py : new constant
* src/order/tests.py : new unit test

### Status

### How to verify this change
- In Souschef, Create a few orders for a date YYYY-MM-DD
- In Souschef, Orders / Reset / View (eye icon) / Pull down arrow at top right : Change the status of one of these orders for date YYYY-MM-DD to something different from "Ordered"
- In Souschef, Orders / Reset and notice which orders have status "Ordered" for date YYYY-MM-DD
- In the docker bash shell, run : ``` $ python3 manage.py setordersdelivered YYYY-MM-DD```
- In Souschef, Orders / Reset and notice that the status of orders that had status "Ordered" for date YYYY-MM-DD is now "Delivered".
